webapp/src/blocks.tsx

```diff
@@ -990,6 +1001,60 @@ export class Editor extends toolboxeditor.ToolboxEditor {
         };
     }
 
+    /**
+     * Provides a user-facing explanation of the errors in the error list.
+     */
+    async getErrorHelp() {
+        const code = this.serializeBlocks(false, true /* forceKeepIds */);
+        try {
+            const helpResponse = await getErrorHelpAsTour(this.errors, "blocks", code);
+            const tour = this.createTourFromResponse(helpResponse);
+            this.parent.showTour(tour);
+        } catch (e) {
+            pxt.reportException(e);
+
+            if (e instanceof ErrorHelpException) {
+                core.errorNotification(e.getUserFacingMessage());
+            } else {
+                core.errorNotification(lf("Sorry, something went wrong. Please try again later."));
```

Comment:
Suggest removing "Sorry".

```suggestion
                core.errorNotification(lf("Something went wrong. Please try again later."));
```